.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,97 @@
+name: Docker
+
+on:
+  push:
+    branches: [ "main" ]
+    paths-ignore:
+      - README.md
+
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  # Use docker.io for Docker Hub if empty
```

Comment:
question (non-blocking): does this comment mean to say that the fallback default is docker hub? Might be unclear but could just be me.

Reply:
The fallback if env.REGISTRY isn't set - it's leftover from the example actions code I copied, I agree it's unclear and I'm gonna remove it (I should have done that before you reviewed since I think it'll dismiss your approval)

Reply:
I keep forgetting that non-blocking comments are in fact somewhat blocking, sorry.

Reply:
heh - only if I decide to deal with them in this PR vs a follow-on issue (or, not deal with them at all), so I think they're still valuable to speak up about in an approval.

Reply:
I might be able to put stuff like that in one-off comments rather than reviews--can try that next time.